config.py

```diff
@@ -14,6 +14,9 @@ class Config(object):
     SECURITY_USER_NAME = os.getenv('SECURITY_USER_NAME')
     SECURITY_USER_PASSWORD = os.getenv('SECURITY_USER_PASSWORD')
     BASIC_AUTH = (SECURITY_USER_NAME, SECURITY_USER_PASSWORD)
+    OAUTH_CLIENT_ID = os.getenv('OAUTH_CLIENT_ID')
```

Comment:
Think these need changed to username and password.

Reply:
@gemmai95 I think we're safe with the OAUTH / DJANGO stuff. Still seems to be CLIENT_ID, etc.